layouts/partials/flex/body-aftercontent.html

```diff
@@ -9,7 +9,7 @@
 
   <div class="utterances-wrapper">
     <script src="/js/utterances/client.js"
-            repo="docs/docs"
+            repo="alebabai/docs"
```

Comment:
Oops

Reply:
yeap, my debug stuff - removed it already)